src/Search/index.js

```diff
@@ -114,48 +153,20 @@ const Search = ({ searchIndex = {}, placeholder = 'Search…', ...props }) => {
         }}
         onSubmit={() => {
           if (results.length > 0) {
-            navigate(results[0].path)
+            navigate(withPrefix(results[0].path))
           }
         }}
+        autoComplete='off'
       />
       <Popover
         isOpen={isOpen}
-        style={{
-          position: 'absolute',
-          left: '0px',
-          top: '32px',
-          zIndex: '1000',
-          width: '325px'
-        }}
+        css={css`
+          position: absolute;
+          top: var(--spectrum-global-dimension-size-800);
+          width: var(--spectrum-global-dimension-size-4600);
```

Comment:
The search popoever is supposed to have a width of 325px and a max height of 424px.